gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/JavaServiceDistributionPlugin.java

```diff
@@ -138,16 +138,23 @@ public void apply(Project project) {
             task.setMainClassName(distributionExtension.getMainClass().get());
             task.setApplicationName(distributionExtension.getDistributionServiceName().get());
             task.setDefaultJvmOpts(distributionExtension.getDefaultJvmOpts().get());
+            task.dependsOn(manifestClassPathTask);
```

Comment:
we should only add the dependency if `enabledManifestClasspath` is set

Reply:
this is how `distTar` currently handles it, and the `manifestClassPathTask` no-ops if that flag isn't set.